cloud-deploy/src/utils/cloud-deploy.schema.json

```diff
@@ -101,6 +101,16 @@
               "description": "Deploy revision with or without traffic",
               "type": "boolean",
               "default": true
+            },
+            "static-egress-ip": {
+              "description": "Use the NAT router when making external requests",
+              "type": "boolean",
+              "default": true
+            },
+            "direct-vpc-connection": {
+              "description": "Whether to use the vpc connector or direct connection to the vpc",
+              "type": "boolean",
+              "default": false
```

Comment:
Is it backwards compatible to default to false?

Reply:
yes there are 3 booleans that has an effect on the outcome of this

(default)
the old vpc-connector: (true) | false = 1
direct-vpc-connection: true | (false) = 2
static-egress-ip: (true) | false = 3

   1       2      3
false false false:  == private traffic via direct vpc on old k8s-subnet 
false false true:   == all traffic via direct vpc on old k8s-subnet
false true  true:   == all traffic via direct vpc on new subnet (cloudrun-subnet)
true  true  true:   == all traffic via direct vpc on new subnet (cloudrun-subnet)
true  true  false:  == private traffic via direct vpc on new subnet
true  false false:  == private traffic via vpc connector
true  false true:   == ( todays default setup ) all traffic via vpc connector